### ISSUE TITLE
False positive in loops

### ISSUE THREAD

**Issue by Antonboom** [sanposhiho/wastedassign#26]
Hello!

```golang
	var i int
	for res := range results {
		suite.requireEqualResults(expected[i], res)
		i++ // reassigned, but never used afterwards (wastedassign)
	}
```

**Comment by sanposhiho** [comment]
Hi @Antonboom .
This issue is solved in [v0.2.0](https://github.com/sanposhiho/wastedassign/releases/tag/v0.2.0).

If you use golangci-lint, please wait for [this PR](https://github.com/golangci/golangci-lint/pull/1815) to be merged and updated to the new version

Thank you for your report.

**Comment by Antonboom** [comment]
Thank you!
